sysutils/snapraid/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        amadvance snapraid 12.3 v
+
+name                snapraid
```

Comment:
`name` and `version` are implicitly set by `github.setup`, so you can delete the lines specifying `name` and `version` since `githup.setup` sets those already.